docs/reference/triggers/kinesis.md

```diff
@@ -28,7 +28,7 @@ triggers:
       secretAccessKey: "my-secret"
       regionName: "eu-west-1"
       streamName: "my-stream"
-      shards: [shard-0, shard-1, shard-2]
+      shards: [shardId-000000000000,shardId-000000000001,shardId-000000000002,shardId-0000000000003]
```

Comment:
Following [Xingxeng's comment](https://iguazio.atlassian.net/browse/ML-8393?focusedCommentId=162509), I think we should keep the example as is, and write a note below that says:

If Kinesis is configured with On-Demand capacity mode, the shard ids should be set as:
`[shardId-000000000000,shardId-000000000001,shardId-000000000002,...]`